### PR TITLE
Fixed some things in the Password Gen

### PR DESCRIPTION
**Pull request by DeathByKorea**
/Fixed it not loading password on open
/Changed how easter egg thing is triggered (click on label)
/Some other optimization goodness

### CODE REVIEW

**Comment by DeathByKorea** [comment]
Also, I am extremely bored right now, if you're wondering why the hell I just did that.

**Comment by tavofigse** [comment]
Vidrioso

**Comment by francocorreasosa** [comment]
Vidrioso eh

**Comment by francocorreasosa** [comment]
Mergeado por lastima

File: PasswordGenerator/PasswordGenerator/Form1.vb [hunk 1]
```diff
@@ -1,10 +1,18 @@
 ﻿Imports System.Text
 Public Class Form1
-    Public Const PI As Double = 3.141519
```
Comment (francocorreasosa) [inline PasswordGenerator/PasswordGenerator/Form1.vb]: I knew that someday someone would remove this for me. Thanks

**Comment by GonzaloRizzo** [comment]
@DeathByKorea How the hell did you find this project?

**Comment by francocorreasosa** [comment]
@GonzaloRizzo lame troll, banned

**Comment by DeathByKorea** [comment]
Was just browsing vb projects because why not. @gonzalorizzo

**Comment by francocorreasosa** [comment]
@DeathByKorea Appreciate it.

**Comment by GonzaloRizzo** [comment]
@DeathByKorea You are a great man

**Comment by francocorreasosa** [comment]
@GonzaloRizzo I thought I banned you.

**Pull request merged**